climakitae/data/variable_catalog.csv

```diff
@@ -1,49 +1,52 @@
 wspd10mean,mean 10-meter wind speed,daily/monthly,m/s,Mean wind speed at 10m,The average wind speed (magnitude) at 10m above the Earth's surface. ,ae_orange,FALSE
-etrans_sfc,evapotranspiration at the surface,daily/monthly,mm/d,Evapotranspiration,"The physical process by which liquid water in ground/surface enters the atmosphere, including evaporation and plant transpiration. By convention, negative evapotranspiration is condensation, in which a liquid forms. ",ae_diverging,FALSE
+wspd10max,maximum 10-meter wind speed,daily/monthly,m/s,Maximum wind speed at 10m,The maximum wind speed at 10m above the Earth's surface. ,ae_orange,TRUE
+psfc,surface pressure,daily/monthly,hPa,Surface Pressure,Air pressure at Earth's surface.,ae_diverging,TRUE
+q2,2-meter specific humidity,daily/monthly,g/kg,Specific humidity at 2m,"The ratio of the mass of water vapor over the mass of total (moist) air, measured 2m above Earth's surface.",ae_blue,TRUE
+tskin,surface skin temperature ,daily/monthly,K,Surface skin temperature,Temperature of the Earth's surface (ground surface).,ae_orange,TRUE
+t2min,daily minimum 2-m temperature ,daily/monthly,K,Daily minimum air temperature at 2m,The minimum daily air temperature at 2m above the Earth's surface. ,ae_orange,TRUE
+t2max,daily maximum 2-m temperature ,daily/monthly,K,Daily maximum air temperature at 2m,The maximum daily air temperature at 2m above the Earth's surface. ,ae_orange,TRUE
+lw_dwn,downwelling longwave flux at bottom,daily/monthly,W/m2,Instantaneous downwelling longwave flux at bottom,Longwave radiation emitted by Earth's surface that is reflected back downwards by clouds. ,ae_orange,TRUE
 sw_dwn,downwelling shortwave flux at bottom,daily/monthly,W/m2,Instantaneous downwelling shortwave flux at bottom,"Shortwave radiation from the sun that reaches Earth's surface, minus radiation reflected back to space by clouds or absorbed by the atmosphere.",ae_orange,TRUE
-t2,2-meter temperature ,daily/monthly,K,Air Temperature at 2m,Temperature of the air 2m above Earth's surface. This is the measure of air temperature used for most modeling applications.,ae_orange,TRUE
+sw_sfc,shortwave flux at the surface,daily/monthly,W/m2,Shortwave flux at the surface,Shortwave radiation from the sun that reaches Earth's surface. ,ae_orange,TRUE
+lw_sfc,longwave flux at the surface,daily/monthly,W/m2,Longwave flux at the surface,Longwave radiation emitted by Earth's surface.,ae_orange,TRUE
+sh_sfc,sensible heat flux at the surface,daily/monthly,W/m2,Sensible heat flux at the surface,"The movement of energy generated by a difference in temperatures between the surface and the atmosphere. By convention, positive sensible heat flux is directed from the surface up into the atmosphere. ",ae_diverging,TRUE
+prec_snow,snow precipitation,daily/monthly,mm,Snowfall,Portion of the total precipitation in the form of snow. ,ae_blue,TRUE

```

Comment:
Confirming that for monthly/daily snowfall that it is just the snow portion with no ice; whereas for hourly it is snowfall with both snow and ice?

Reply:
Will follow up again with Brian